src/App.php

```diff
@@ -1193,4 +1224,82 @@ private function getRenderedPortals(): array
 
         return $portals;
     }
+
+    /**
+     * Return true if $_GET[$key] exists.
+     */
+    public function issetRequestGetParam(string $key): bool
+    {
+        return isset($this->request->getQueryParams()[$key]);
+    }
+
+    /**
+     * Return whole $_GET array data.
+     */
+    public function getRequestGetParams(): array
+    {
+        return $this->request->getQueryParams();
+    }
+
+    /**
+     * Return $_GET param by key or null if not exists.
+     *
+     * @return mixed
```

Comment:
AFAK the GET can be string, null

and array of strings, but we do not want to support it probably using this method, as longer alternative exists :)

Reply:
test cases:

```
var_dump($app->hasRequestGetParam('x')); // test URL: ?x, passed, empty query value (without '=') is still non-null/string
var_dump($app->getRequestGetParam('x')); // test URL: ?x[]=1&x[]=2, TypeError: Return value of Atk4\Ui\App::getRequestGetParam() must be of the type string, array returned
```